libs/ui/src/lib/theme.tsx

```diff
@@ -1,11 +1,22 @@
-import {createTheme, CSSObject, SimplePaletteColorOptions} from '@mui/material'
+import {
+  createTheme,
+  CSSObject,
+  SimplePaletteColorOptions,
+  Theme as MaterialTheme
+} from '@mui/material'
 import {createBreakpoints} from '@mui/system'
 
 const {
   palette: {augmentColor},
   breakpoints: originalBreakpoints
 } = createTheme()
 
+declare module '@emotion/react' {
```

Comment:
This is the magic that allows us to use the emotion styled now, which fixes our SWC problem